src/new-client/.env

```diff
@@ -1,8 +1,5 @@
-NODE_ENV=development
-REACT_APP_ENV=default
-REACT_APP_VERSION=$npm_package_version
-REACT_APP_NAME=$npm_package_name
+VITE_VERSION=$npm_package_version
+VITE_NAME=$npm_package_name
 REACT_EDITOR=code
-REACT_APP_GREENKEY_URL=ws://localhost:8888/client/ws/speech
+VITE_GREENKEY_URL=ws://localhost:8888/client/ws/speech
```

Comment:
I don't think so, but the point of this was to make it compatible with vite and this is how we pass the env variables with vite, so just in case I changed it.